corehq/apps/export/views/utils.py

```diff
@@ -386,7 +386,8 @@ def clean_odata_columns(export_instance):
     for table in export_instance.tables:
         for column in table.columns:
             column.label = column.label.replace('@', '').replace(
-                '.', ' ').replace('\n', '').replace('\t', ' ').replace('#', '')
+                '.', ' ').replace('\n', '').replace('\t', ' ').replace(
+                '#', '').replace(',', '')
```

Comment:
how do we choose between spaces and empty strings here? seems a little arbitrary.

Reply:
typically a comma is followed by a space. so it makes sense to replace them with an empty string. the period normally is the same, but by default the label names are always `form.questionid` or `form.groupname.questionid`---hence the choice to use a space in this case.